src/Pages/Login.jsx

```diff
@@ -0,0 +1,34 @@
+import AuthForm from "../components/AuthForm/AuthForm";
+import { useEffect, useState } from "react";
```

Comment:
please also add React, otherwise in test files it will show error

Reply:
@ray-wtag bhai, I think we don't need to import react in vite due to its Fast Refresh feature.